app/components/provider_interface/setup_complete_next_steps_component.html.erb

```diff
@@ -8,7 +8,7 @@
   </li>
   <% if user_can_manage_users? %>
     <li>
-      <%= govuk_link_to t('.invite_or_manage_users'), provider_interface_provider_users_path %>
+      <%= govuk_link_to t('.invite_or_manage_users'), provider_interface_organisation_settings_path %>
```

Comment:
We missed this when doing the initial work. Linking to the user page will currently redirect you to the org settings page when the feature flag is on, so this isn't really a change in behaviour